docs/developing-fuzzbench/adding_a_new_benchmark.md

```diff
@@ -161,18 +212,78 @@ export FUZZER_NAME=afl
 export BENCHMARK_NAME=libpng-1.2.56
 
 make build-$FUZZER_NAME-$BENCHMARK_NAME
+
+# This command will fuzz forever. Press Ctrl-C to stop it.
 make run-$FUZZER_NAME-$BENCHMARK_NAME
 ```
 Finally, add your benchmark to the list of OSS-Fuzz benchmarks in
 [test_fuzzer_benchmarks.py](https://github.com/google/fuzzbench/blob/master/.github/workflows/test_fuzzer_benchmarks.py)
 
 This ensures that CI tests your benchmark with all fuzzers.
 
-## Testing the benchmark in CI
+## Submitting the benchmark in a pull request
 
-Add your benchmark to the `STANDARD_BENCHMARKS` list in
+Add your benchmark to a list in
 [build_and_test_run_fuzzer_benchmarks.py](https://github.com/google/fuzzbench/blob/master/.github/workflows/build_and_test_run_fuzzer_benchmarks.py)
-so that it will be tested in CI.
+so that it will be tested in CI. If your benchmark is a standard benchmark, add
+it to the `STANDARD_BENCHMARKS` list, otherwise add it to the the
```

Comment:
Why do we need two different lists now, can we have a single list ?

Reply:
This is only for parallelizing builds so they happen faster.